skiko/src/commonMain/kotlin/org/jetbrains/skia/Bitmap.kt

```diff
@@ -564,16 +560,18 @@ class Bitmap internal constructor(ptr: NativePointer) : Managed(ptr, _FinalizerH
         return try {
             _imageInfo = null
             Stats.onNativeCall()
-            _nInstallPixels(
-                _ptr,
-                info.width,
-                info.height,
-                info.colorInfo.colorType.ordinal,
-                info.colorInfo.alphaType.ordinal,
-                getPtr(info.colorInfo.colorSpace),
-                pixels,
-                rowBytes
-            )
+            interopScope {
```

Comment:
To avoid unneeded memory allocations we probably shall make interopScope expect, and actualize it as 

```
object theScope: InteropScope
inline fun <T> interopScope(block: InteropScope.() -> T): T {
       return theScope.block()
    }
}
```